lxl-web/src/lib/components/my-pages/Libraries.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+	import { page } from '$app/stores';
```

Comment:
`$app/stores` is deprecated in the version of svelteKit we use now (I know it's all over the old components), so when creating new stuff we should use `$app/state` instead and then just change `$page` to `page`...
https://svelte.dev/docs/kit/migrating-to-sveltekit-2#SvelteKit-2.12:-$app-stores-deprecated